Citavi 5 XML.js

```diff
@@ -449,12 +473,45 @@ async function doImport() {
 	const totalProgress = references.length + tasks.length + categories.length;
 	const progress = { total: totalProgress * 2, current: 0 };
 
-	await importItems({ references, doc, citaviVersion, rememberTags, itemIdList, progress, unfinishedReferences });
+	await importItems({ references, doc, citaviVersion, rememberTags, rememberCustomFields, itemIdList, progress, unfinishedReferences });
 	await importUnfinished({ doc, itemIdList, unfinishedReferences, progress });
 	await importTasks({ tasks, progress });
+	await importStandalonNotes(doc, rememberTags, progress);
 	importCategories({ categories, doc, progress });
 }
 
+function extractNote(doc, noteXML, rememberTags) {
+	var noteObject = {};
+	noteObject.id = ZU.xpathText(noteXML, '@id');
+	var title = ZU.xpathText(noteXML, 'CoreStatement');
+	var text = ZU.xpathText(noteXML, 'Text');
+	var pages = extractPages(ZU.xpathText(noteXML, 'PageRange'));
+	noteObject.note = '';
+	if (title) {
+		noteObject.note += '<h1>' + title + "</h1>\n";
+	}
+	if (text) {
+		text = text.split(/\r?\n/).join("<br />");
+		noteObject.note += "<p>" + text + "</p>\n";
+	}
```

Comment:
Add text to note if address is file name. Empty line before text to ease inserting of image later. 
```suggestion
	}
	if (address) {
		var addressMap = JSON.parse(address);
		// Based on Demo project mapping is:
		// 1->File
		// 5->Link
		if (addressMap.LinkedResourceType==1){
			noteObject.note += "<p></p><p>" + '<strong>Missing Image with filename:</strong> </p>';
			noteObject.note += "<pre><code>" +  addressMap.UriString + "</code></pre>\n";
			var hasMissingImage = true;
		}
	}
```